<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TRPL-PUN-0125" failures="0" tests="33" name="demoLinkLogin.DeptStatutory" time="184.511" errors="2" timestamp="2024-10-31T18:14:53 IST" skipped="31">
  <testcase classname="demoLinkLogin.DeptStatutory" name="DepartmentSummaryFinanceStatutory" time="184.504">
    <error message="java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
Build info: version: &#039;3.141.0&#039;, revision: &#039;2ecb7d9a&#039;, time: &#039;2018-10-31T20:09:30&#039;
System info: host: &#039;TRPL-PUN-0125&#039;, ip: &#039;192.168.31.67&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.9&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
Build info: version: '3.141.0', revision: '2ecb7d9a', time: '2018-10-31T20:09:30'
System info: host: 'TRPL-PUN-0125', ip: '192.168.31.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.9'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:489)
at demoLinkLogin.DeptStatutory.DepartmentSummaryFinanceStatutory(DeptStatutory.java:1148)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 31 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
at java.base/java.net.Socket.connect(Socket.java:633)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 51 more
]]>
    </error>
  </testcase> <!-- DepartmentSummaryFinanceStatutory -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="browserClosing" time="0.005">
    <error message="java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
Build info: version: &#039;3.141.0&#039;, revision: &#039;2ecb7d9a&#039;, time: &#039;2018-10-31T20:09:30&#039;
System info: host: &#039;TRPL-PUN-0125&#039;, ip: &#039;192.168.31.67&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.9&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
Build info: version: '3.141.0', revision: '2ecb7d9a', time: '2018-10-31T20:09:30'
System info: host: 'TRPL-PUN-0125', ip: '192.168.31.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.9'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:452)
at login.BasePage.closeBrowser(BasePage.java:345)
at demoLinkLogin.DeptStatutory.browserClosing(DeptStatutory.java:2887)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)
at org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:779)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:755)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:12908
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 38 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
at java.base/java.net.Socket.connect(Socket.java:633)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 58 more
]]>
    </error>
  </testcase> <!-- browserClosing -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="Entities" time="0.000">
    <skipped/>
  </testcase> <!-- Entities -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="LocationCount" time="0.000">
    <skipped/>
  </testcase> <!-- LocationCount -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="DepartmentCountMatch" time="0.000">
    <skipped/>
  </testcase> <!-- DepartmentCountMatch -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="CompliancesCountMatch" time="0.001">
    <skipped/>
  </testcase> <!-- CompliancesCountMatch -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="UsersCountMatch" time="0.000">
    <skipped/>
  </testcase> <!-- UsersCountMatch -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="SummaryofOverdueCompliances" time="0.000">
    <skipped/>
  </testcase> <!-- SummaryofOverdueCompliances -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="NotCompleted_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- NotCompleted_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="ClosedDelayed_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedDelayed_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="ClosedTimely_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedTimely_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="NotApplicable_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- NotApplicable_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="Overdue_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- Overdue_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="pendingForReview_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- pendingForReview_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="inProgress_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- inProgress_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="rejected_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- rejected_PieChart -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="RiskSummaryCriticalStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryCriticalStatutory -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="RiskSummaryHighStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryHighStatutory -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="RiskSummaryMediumStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryMediumStatutory -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="RiskSummaryLowStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryLowStatutory -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="NotCompleted_PieChartPeriod" time="0.001">
    <skipped/>
  </testcase> <!-- NotCompleted_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="ClosedDelayed_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedDelayed_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="ClosedTimely_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedTimely_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="NotApplicable_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- NotApplicable_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="Overdue_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- Overdue_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="dueToday_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- dueToday_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="pendingForReview_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- pendingForReview_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="Inprogress_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- Inprogress_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="rejected_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- rejected_PieChartPeriod -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="complianceCalendar" time="0.000">
    <skipped/>
  </testcase> <!-- complianceCalendar -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="DetailedReport" time="0.000">
    <skipped/>
  </testcase> <!-- DetailedReport -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="AssignmentReport" time="0.000">
    <skipped/>
  </testcase> <!-- AssignmentReport -->
  <testcase classname="demoLinkLogin.DeptStatutory" name="ComplianceDocuments" time="0.000">
    <skipped/>
  </testcase> <!-- ComplianceDocuments -->
</testsuite> <!-- demoLinkLogin.DeptStatutory -->
